_rules/html-page-lang-xml-lang-match-5b7ae0.md

```diff
@@ -57,23 +57,23 @@ Since most assistive technologies will consistently use `lang` over `xml:lang` w
 
 #### Passed Example 1
 
-`html` element with matching primary language subtags for `lang` and `xml:lang`.
+This `html` element has identical [primary language subtags][] for its `lang` and `xml:lang` attributes.
 
 ```html
 <html lang="en" xml:lang="en"></html>
 ```
 
 #### Passed Example 2
 
-`html` element with matching primary and extended language subtags for `lang` and `xml:lang`.
+This `html` element has identical [primary language subtags][] for its `lang` and `xml:lang` attributes. The [extended language subtags][] also match.
```

Comment:
Is there a reason for not keeping this as a single sentence?

This `html` element has identical [primary language subtags][] and [extended language subtags][] for its `lang` and `xml:lang` attributes.

Reply:
I felt that it was closer to the "[test target] [matches expectation]" format. Here, the expectation is about the primary tag. The second sentence has nothing to with the expectation and is additional information and remark. That's why I think it is better suited as a separate sentence, keeping the first one strictly on the expectation.

Reply:
So is the subtags matching is incidental to the success criteria and rule requirement? Wondering if "Incidentally" should be added to the second sentence, or if Passed Example 3 alone would be sufficient if the second sentence read "The extended language subtags are not required to match."

Reply:
Probably also mostly conversation for a different PR.

Reply:
It is incidental for the rule. It is less clear for the SC…
There has been some discussion around it (can't find the issue/PR) between @kasperisager and @dd8 if I remember correctly.
Essentially, I remember that secondary tags don't really cause accessibility issues. E.g., if a page is written in British but with a `lang="en-US"`, that's not going to be a problem. So, similarly for matching `lang`/`xml:lang`, if one of them says "en-US" and the other "en-UK", that's not going to be a real problem.

One notable case where this would cause accessibility issue is between Cantonese (`zh-yue`) and Mandarin (`zh-cmn`) which are written the same way but pronounced very differently (to the point that speakers don't understand each other). So the wrong subtag would cause screen readers to output incomprehensible speech.

Given that this was the only case we could find where the subtag really matters for accessibility, the rule only focus on primary tags and there is a note in Expectation mentioning this.

It is not clear whether WCAG require perfect identification of subtags also…

I will nonetheless update the description to make this a bit more obvious.